From f423cac4d8c5cd41085b49730d5b6f7d5a21a437 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Tue, 11 Mar 2025 06:34:37 +0100
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index de4ef6e..8b36395 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -22,7 +22,7 @@ classifiers = [
     "Programming Language :: Python :: 3.12",
 ]
 dependencies = [
-    "tensorflow<2.17",
+    "tensorflow<=2.17",
     "tf_keras",
     "scipy",
     "matscipy",
